src/models/chan.js

```diff
@@ -95,6 +95,7 @@ Chan.prototype.getMode = function(name) {
 
 Chan.prototype.toJSON = function() {
 	var clone = _.clone(this);
+	clone.users = [];
```

Comment:
Maybe a quick comment here just mentioning why we do this. I can see it would be confusing for people coming to it.